tool/state-processor/src/staking/mod.rs

```diff
@@ -154,16 +154,21 @@ impl<S> Processor<S> {
 		log::info!("`total_deposit({total_deposit})`");
 		log::info!("` total_staked({total_staked})`");
 
-		log::info!("set `Staking::RingPool` and `Staking::KtonPool`");
-		self.shell_state.insert_value(b"Staking", b"RingPool", "", ring_pool).insert_value(
-			b"Staking",
+		log::info!("set `DarwiniaStaking::RingPool` and `DarwiniaStaking::KtonPool`");
+		self.shell_state.insert_value(b"DarwiniaStaking", b"RingPool", "", ring_pool).insert_value(
+			b"DarwiniaStaking",
```

Comment:
![image](https://user-images.githubusercontent.com/11801722/223956001-728e68f1-e755-48a4-a90e-8acde94062e9.png)

I think we need to update these constants  to make CI happy.